site/src/main/java/uk/nhs/digital/ps/beans/navigation/CiBreadcrumbProvider.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * <p>
+ * Default Hippo Breadcrumb plugin uses menu/folder structure to generate breadcrumbs,
+ * but this doesn't fit for clinical indicator content (ciLanding, publication,
+ * series, datasets) as they have a very specific setup in the CMS.
+ * </p>
+ */
+public class CiBreadcrumbProvider  {
+
+    private static final Logger log = LoggerFactory.getLogger(CiBreadcrumbProvider.class);
+    private static final String PATH_CI_LANDING_PAGES = "publication-system/ci-hub/";
+    private static final String SEPARATOR = "/";
+
+    private static final Map<String, String> ciFolderToCiLandingMap = new HashMap<String, String>() {
+        {
+            put("ccg-outcomes-indicator-set", "ccg-outcomes-indicator-set");
+            put("compendium-of-population-health-indicators", "compendium-indicators");
+            put("adult-social-care-outcomes-framework-ascof", "social-care");
+            put("nhs-outcomes-framework", "nhs-outcomes-framework");
+            put("seven-day-services", "seven-day-services");
+            put("shmi", "summary-hospital-level-mortality-indicator-shmi");
+            put("uk-gtn", "uk-gtn");
+        }
+    };
+
+    public CiBreadcrumb getBreadcrumb(final HstRequest request) {
+
+        final HstRequestContext ctx = request.getRequestContext();
+        HippoBean content = ctx.getContentBean();
+        boolean isClinicalIndicatorDocument = false;
+        List<BreadcrumbItem> ciBreadcrumbItems = new ArrayList<>();
+
+        // Is this a CiLanding page?
+        if (ciFolderToCiLandingMap.containsValue(content.getName())) {
+            isClinicalIndicatorDocument = true;
+        } else {
+            // Traverse the hierarchy to see if this document sits within the Clinical Indicator
+            // folders, and if so load the appropriate ciLanding page in order to create the
+            // navigation link. Provides that extra navigation to give user more accurate hierarchy
+            HippoFolder folder = (HippoFolder) content.getParentBean();
+            while (!HippoBeanHelper.isRootFolder(folder)) {
+
+                String folderName = folder.getName();
+
+                if (ciFolderToCiLandingMap.containsKey(folderName)) {
+                    final HippoBean base = RequestContextProvider.get().getSiteContentBaseBean();
+                    final HippoBean ciLandingPage = base.getBean(
+                        PATH_CI_LANDING_PAGES + ciFolderToCiLandingMap.get(folderName));
+
+                    isClinicalIndicatorDocument = true;
+                    ciBreadcrumbItems.add(createBreadcrumbItem(ctx, ciLandingPage));
+                    break;
+                }
+                folder = (HippoFolder) folder.getParentBean();
+            }
+
+            // If Publication or dataset, find parent links for breadcrumb
+            // (note: not required for series since series
+            // documents are already top level breadcrumb)
+            if (content.getClass().equals(Publication.class)) {
```

Comment:
Can you change to `content instanceof Publication`

Reply:
Updated